packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2894,8 +2896,8 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
-  /// If non-null, sets the "hasCheckedState" semantic to true and the
-  /// "isChecked" semantic to the given value.
+  /// If non-null, sets the [hasCheckedState] semantic to true and the
+  /// [isChecked] semantic to the given value.
```

Comment:
These should point to the actual place where those exist (they don't exist on this class).